scripts/zones/Boneyard_Gully/mobs/Shikaree_X_HW.lua

```diff
@@ -85,23 +85,27 @@ entity.onMobWeaponSkill = function(target, mob, skill)
     local skillID     = skill:getID()
     local battlefield = mob:getBattlefield()
 
-    -- Solo skill message when only one Shikaree is alive (scState 4 = SOLO)
-    if battlefield and battlefield:getLocalVar('scState') == 4 then
-        mob:messageText(mob, ID.text.SHIKAREE_X_OFFSET + 4)
+    if not battlefield then
+        return
     end
 
     -- 2-Hour message.
     if skillID == xi.mobSkill.FAMILIAR_1 then
         mob:messageText(mob, ID.text.SHIKAREE_X_2HR)
     end
 
+    -- Solo skill message when only one Shikaree is alive (scState 4 = SOLO)
+    if battlefield:getLocalVar('scState') == 4 then
+        mob:messageText(mob, ID.text.SHIKAREE_X_OFFSET + 4)
+    end
```

Comment:
If the mithra is solo and she 2-hours, this will still print 2 messages. 1 for being a 2-hour. And then another for being solo. I would guess It would only print 1?

Use an elseif if so.